altimeter.h

```diff
@@ -46,6 +46,11 @@ class Altimeter {
     }
   }
 
+  void setSeaLevel(float currSeaLevel) {
```

Comment:
I would mark this `noexcept`

Reply:
This still hasn't been addressed, but I'm not going to hold up the PR over this. Otherwise LGTM